HALO/wales/main.yaml

```diff
@@ -0,0 +1,35 @@
+sources:
+  cloudparameter:
+    args:
+      path: "{{CATALOG_DIR}}/wales_cloudtop.yaml"
+    description: 'WALES Lidar cloud top heights'
+    driver: intake.catalog.local.YAMLFileCatalog
+    metadata: {}
+
+  wv:
+    args:
+      path: "{{CATALOG_DIR}}/wales_wv.yaml"
+    description: 'WALES Lidar Water Vapour Mixing Ratio in g/kg'
```

Comment:
The data in one file (HALO-0205) contains the following attributes in the variable `humidity_mixing_ratio`:
```
Attributes:
    long_name:  water vapor volume mixing ratio
    units:      ppm
```
This suggests that the data is
* the volume mixing ratio and not the mass mixing ratio
* the units are parts per million and not g/kg i.e. parts per thousand

Reply:
good point. I looked at a different source for information. But this seems more plausible to look at the netCDF file directly :D